### DISCUSSION TITLE
May be a bug and how to fix it

### DISCUSSION THREAD

**Discussion by Gr-Nobel** [mdelobelle/metadatamenu#205] [General]
![image](https://user-images.githubusercontent.com/95166364/206866301-2a0756ae-5822-47c7-a859-9206fea1c15f.png)
i tried to fix all the files containing this fileClass.
How to spot the problem. 
in which page

**Comment by Gr-Nobel** [comment]
what is item.trim @mdelobelle

**Reply by mdelobelle** [reply]
Hi this seems to be related to the definition of one Multi File field.
Can you paste the content of this fileClass note?

**Reply by Gr-Nobel** [reply]
Fazr:: {"type":"Multi","options":{"valuesList":{"1":"54 2 Rakah ফরজ জামাতে","2":"27 1 Rakah ফরজ জামাতে","3":"2 Rakah সুন্নত","4":"0 Rakah সুন্নত ","5":"2 Rakah মসজিদে আদায়","6":"2 Rahkah বাসায় আদায়","7":"2 Rakah কাজায়","8":"0 কাজায়ও আদায় করি নি"},"sourceType":"ValuesList","valuesListNotePath":"","valuesFromDVQuery":""}}

Zuhur:: {"type":"Multi","options":{"valuesList":{"1":"4 Rakah সুন্নত","2":"0 Rakah সুন্নতBefore","3":"108   4 Rakah ফরজ জামাতে","4":"81 3 Rakah ফরজ জামাতে","5":"54  2 Rakah ফরজ জামাতে","6":"27 1 Rakah ফরজ জামাতে","7":"2 রাকাত সুন্নত","8":"0 Rakah সুন্নতAfter","9":"40   4 Rakah মসজিদে","10":"4 Rakah বাসায় আদায়","11":"0 কাজা আদায়"},"sourceType":"ValuesList","valuesListNotePath":"","valuesFromDVQuery":""}}

Asr:: {"type":"Multi","options":{"valuesList":{"1":"4 Rakah সুন্নত","2":"0 Rakah রাকাত সুন্নত","3":"108 - 4 Rakah ফরজ জামাতে","4":"81 3 Rakah ফরজ জামাতে","5":"54  2 Rakah ফরজ জামাতে","7":"27  1 Rakah ফরজ জামাতে","8":"40  4 Rakah মসজিদে","9":"4 Rakah বাসায় আদায়","10":"0 কাজা আদায়"},"sourceType":"ValuesList","valuesListNotePath":"","valuesFromDVQuery":""}}

Magrib:: {"type":"Multi","options":{"valuesList":{"1":"81 3 Rakah জামাতে","2":"54  2 Rakah জামাতে","3":"27  1 Rakah জামাতে","4":"2 Rakah সুন্নত","5":"0 Rakah সুন্নত","6":"3 Rakah বাসায় আদায়","7":"3 Rakah মসজিদে","8":"0 Rakah কাজা আদায়"},"sourceType":"ValuesList","valuesListNotePath":"","valuesFromDVQuery":""}}
Esha:: {"type":"Multi","options":{"valuesList":{"1":"4 Rakah সুন্নত","2":"108 - 4 Rakah ফরজ জামাতে","3":"81  3 Rakah ফরজ জামাতে","4":"54  2 Rakah ফরজ জামাতে","5":"27  1 Rakah ফরজ জামাতে","6":"2 রাকাত সুন্নত","7":"0 Rakah সুন্নত","8":"40   4 Rakah মসজিদে","9":"4 Rakah বাসায় আদায়","10":"0 কাজা আদায়"},"sourceType":"ValuesList","valuesListNotePath":"","valuesFromDVQuery":""}}


Witr:: {"type":"Multi","options":{"valuesList":{"1":"30  3 Rahat বাসায়","2":"15  3 Rakah মসজিদে","3":"0 আদায় করি নি"},"sourceType":"ValuesList","valuesListNotePath":"","valuesFromDVQuery":""}}

**Reply by Gr-Nobel** [reply]
> Hi this seems to be related to the definition of one Multi File field. Can you paste the content of this fileClass note?

It's too long sorry . @mdelobelle
